### ISSUE TITLE
JavaScript Evaluation With CSS3 Selectors

### ISSUE THREAD

**Issue by PatrickMurray** [less/less.js#1443]
I'm trying to write a stylesheet that will style links that point to the current page. My code looks something like this:

```
a[href $= "`window.location.pathname`"] {
    color: red;
    }
```

Unfortunately, this does not seem to work;  However, when I replace `window.location.pathname` with the path (lets say "index.html") it works... strange.

I believe this is a bug within the JavaScript evaluation pertaining to CSS3 selectors.  The JavaScript works as proven in the following example:

```
a[href]:before {
    content: "`window.location.pathname`";
    }
```

**TLDR: JavaScript evaluation works, but not in CSS3 selectors :(**

Thanks,
Pat
